Formula/Pelikan.rb

```diff
@@ -0,0 +1,23 @@
+class Pelikan < Formula
+  desc "Pelikan: production-ready cache services"
+  homepage "http://pelikan.io"
+  url "https://github.com/twitter/pelikan/archive/0.1.1.tar.gz"
+  sha256 "98ab698b80e6e7634245e82798a9672b4a0b9f7f603879b398a66a4d97550e7f"
+  head "https://github.com/twitter/pelikan.git"
+
+  depends_on "cmake" => :build
+  depends_on "check" => :build
+
+  def install
+    mkdir "_build"
+    cd "_build" do
+      system "cmake", "..", *std_cmake_args # TODO: adjust args
+      system "make", "-j"
```

Comment:
@bfontaine if compiling from source `make -j` is almost always faster than `make`, because the former uses multiple threads. I do not know if this is kosher with homebrew, but that's what I run when build locally.

Would follow whatever is recommended for homebrew.